java/java-guestbook/kubernetes-manifests/guestbook-mongodb.deployment.yaml

```diff
@@ -0,0 +1,29 @@
+################################################################################
+# WARNING: This MongoDB deployment is not suitable for production as the data is
+# not persistently stored and will go away every time the Pod restarts. Consider
+# using a Helm chart that provisions a StatefulSet instead of Deployment.
+################################################################################
```

Comment:
Why not show that?  (Ok to ignore me here)

Reply:
How to handle data storage was a long discussion, but we decided to prioritize easy deployment and readability over production-readiness for these samples. We thought a comment describing production best practices was a good compromise